mmv1/products/metastore/Service.yaml

```diff
@@ -184,6 +200,32 @@ properties:
       - :DEVELOPER
       - :ENTERPRISE
     default_from_api: true
+    conflicts:
+      - scalingConfig
+
+  - !ruby/object:Api::Type::NestedObject
+    name: 'scalingConfig'
+    description: |
+      Represents the scaling configuration of a metastore service.
+    properties:
+      - !ruby/object:Api::Type::Enum
+        name: 'instanceSize'
+        description: |
+          Metastore instance sizes.
+        required: false
```

Comment:
Should this be marked as required? It will only be required if the scalingConfig block is specified

Reply:
Looks like theres an example without it. How about using exactly_one_of or at_least_one_of here?

It could look like:
exactly_one_of: 
  scaling_config.0.instance_size
  scaling_config.0.scalingFactor

Reply:
> Should this be marked as required? It will only be required if the scalingConfig block is specified

Yes, thanks - it should be required if scalingConfig is there.

Reply:
Since it looks like `exactly_one_of` and `required` are mutually exclusive, I'll go with `exactly_one_of`.